test/unit/spec/common/jobs-Spec.js

```diff
@@ -281,91 +274,166 @@ define(['common/jobs', '/test/data/jobsData'], (Jobs, JobsData) => {
         });
     });
 
+    function generateJobs(jobSpec) {
```

Comment:
I was too lazy to redo the input data nicely => wrote a quick function to do it instead!